includes/Core/Consent_Mode/Consent_Mode.php

```diff
@@ -73,9 +84,12 @@ public function register() {
 			// The `wp_head` action is used to ensure the snippets are printed in the head on the front-end only, not admin pages.
 			add_action(
 				'wp_head',
-				$this->get_method_proxy( 'render_gtag_consent_snippet' ),
+				$this->get_method_proxy( 'render_gtag_consent_data_layer_snippet' ),
 				1 // Set priority to 1 to ensure the snippet is printed with top priority in the head.
 			);
+
+			// Register the consent mode script.
+			add_action( 'wp_enqueue_scripts', fn () => $this->enqueue_script(), 30 );
```

Comment:
Is there a specific reason to use this priority of `30`? I did not notice change in enqueue order when switching to the default value